src/code42cli/cmds/legal_hold.py

```diff
@@ -230,6 +275,24 @@ def _get_all_active_matters(sdk):
     return matters
 
 
+def _get_all_events(sdk, legal_hold_uid, begin_date, end_date, event_type):
```

Comment:
Can we move the filtering logic out into the top-level events command function? That way the name of this function is more accurate.

So it would be like:

```
events = _get_all_events(state.sdk, matter_id, begin, end)
if event_type:
    events = [e for e in events if e["eventType"] == event_type]
if events:
    formatter.echo_formatted_list(events)
```